### DISCUSSION TITLE
Can you please do a 2.8.14.1 build?

### DISCUSSION THREAD

**Discussion by Rygle** [Komet/MediaElch#1425] [General]
https://github.com/Komet/MediaElch/issues/1412

This invisible text bug is really irritating, and I know it is marked as closed/fixed, but it's still the version that's available in the ppa for all debian based distros, and which new Docker images are also based on. I recently made a new Docker image and would have liked to give people the option of the last build, but the ppa makes that hard.

I don't know if there's a way to have multiple builds on the ppa repository. That way if a bug arises, at least another version of the package can be reverted to.

Thanks!

**Comment by bugwelle** [comment]
Hi there,

I plan on doing a releaes this month. Possibly next weekend if time allows for it. I'm also currently setting up a Jenkins pipeline which will do proper Nightly deployments. That works for AppImage + Windows ZIP already. If everything works out, I plan on adding a Nightly PPA as well. But due to limited Debian Packaging knowledge, this will take longer. See https://launchpad.net/~mediaelch/+archive/ubuntu/mediaelch-nightly (currently unused).

Regards,
Andre

**Comment by workflowsguy** [comment]
Please also update the macOS version with the fix for this issue

**Reply by bugwelle** [reply]
Hi,

of course. New releases will be published for all platforms. In the meantime, you could also use the snapshot version provided at https://mediaelch-downloads.ameyering.de/snapshots/macOS/ (just made a new snapshot version just now for macOS).

Regards,
Andre

**Comment by bugwelle** [comment]
MediaElch v2.8.16 is now released https://mediaelch.github.io/mediaelch-blog/posts/mediaelch-v2.8.16/

**Reply by Rygle** [reply]
Thank you!

**Comment by workflowsguy** [comment]
I verified that the issue no longer occurs on macOS.

Thank you!
